<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DependencyView</class>
 <widget class="QWidget" name="DependencyView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="searchLine">
     <property name="placeholderText">
      <string>Search</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="dependencyView">
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionExpandAll">
   <property name="text">
    <string>&amp;Expand All</string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="text">
    <string>&amp;Collapse All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
